hazelcast/src/main/java/com/hazelcast/config/WanReplicationRef.java

```diff
@@ -81,15 +81,35 @@ public WanReplicationRef setMergePolicy(String mergePolicy) {
         return this;
     }
 
+    /**
+     * Add class name implementing the CacheWanEventFilter for filtering WAN replication events.
```

Comment:
I think it's better like "... CacheWanEventFilter or MapWanEventFilter..."
Same for `getFilters` and `setFilters` methods' javadoc.